lib/workflow/reconciler.go

```diff
@@ -29,6 +32,8 @@ type Opts struct {
 	source    string
 }
 
+var minimumPeriodBetweenCreatingPipelineResources = 1100 * time.Millisecond
```

Comment:
```suggestion
var minimumPeriodBetweenCreatingPipelineResources = 1 * time.Second
```
why not 🤔

Reply:
because I'm paranoid

Reply:
it needs to be *atleast* 1 second, so I went slightly above 🤷